composer.json

```diff
@@ -18,9 +19,18 @@
 		"phpcompatibility/phpcompatibility-wp": "2.0.0"
 	},
 	"scripts": {
-		"php:compatibility": "composer install && vendor/bin/phpcs -p -s --runtime-set testVersion '5.2-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
+		"php:compatibility": "composer install && vendor/bin/phpcs -p -s --runtime-set testVersion '5.3-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
```

Comment:
For another PR, I would like to try this without the runtime. The version is set in .phpcs.xml already.